algo.md

```diff
@@ -0,0 +1,15 @@
+1) get requests from 20:00 current timr snd everything after 20:00 yesterday
+2) crate route weight variabe
+varibke priorities
+3) sort by time window... timeWindow function
+4) sort by first come first serve
+5) long itute and latitude, sort by info
+6) use apie to sort beter route whit google api
+7) simple sort equation
+8) reru on weight of priority +1, sloest in time iwindow to next window is last -1
+9) check pi for best routes again
+10) most efficient route
+11) if a for route z 
```

Comment:
Thanks for the algorithm - it would be more readable if we just clean up some of the spelling?